README.md

```diff
@@ -59,13 +59,13 @@ Group folders are handled specially by checking if the username equal `__groupfo
 ### Usage
 
 ```shell
-./solvable_files.sh <data_dir> <db_host> <db_user> <db_pwd> <db_table> <fix,list>
+./solvable_files.sh <data_dir> <mysql|pgsql> <db_host> <db_user> <db_pwd> <db_name> [<fix|list>] [<scan|noscan>]
 ```
 
 ### Output
 
 ```shell
-$ ./solvable_files.sh "$PWD/data/" localhost nextcloud password nextcloud list
+$ ./solvable_files.sh "$PWD/data/" mysql localhost nextcloud password nextcloud
```

Comment:
please add the two other args at the end instead of relying on default values ?
might be easier to understand